main.tf

```
@@ -1,8 +1,9 @@
 module "label" {
-  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.7.0"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
-  namespace = "${var.namespace}"
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.7.0"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  attributes = ["cloudwatch-agent"]
```

```suggestion
  attributes = ["${var.attributes}"]
```

would be better to add a var `attributes` with the default value of `cloudwatch-agent` so it could be changed by the user if needed